server/hotrod/src/main/scala/org/infinispan/server/hotrod/AbstractTopologyAwareEncoder1x.scala

```diff
@@ -21,7 +25,7 @@ abstract class AbstractTopologyAwareEncoder1x extends AbstractEncoder1x with Con
                                                   cfg: Configuration): AbstractHashDistAwareResponse = {
       HashDistAware11Response(lastViewId, serverEndpointsMap, cfg.clustering().hash().numOwners(),
             DEFAULT_CONSISTENT_HASH_VERSION_1x, Integer.MAX_VALUE,
-            cfg.clustering().hash().numVirtualNodes())
+            cfg.clustering().hash().numSegments())
```

Comment:
`numSegments` doesn't work here, it has a completely different meaning from the now unused `numVirtualNodes`. I guess we should just remove `numVirtualNodes` in `HashConfiguration` and replace it with `1` here.